Telemetry/TelemetryEvent.swift

```diff
@@ -85,10 +87,9 @@ public class TelemetryEvent {
         }
 
         if !extras.isEmpty {
-            if value == nil {
-                array.append(nil)
```

Comment:
We recently switched from storing this data in a JSON file to `NSUserDefaults`. I guess we can't store `nil`? @garvankeeley any other suggestions here? Perhaps an empty string instead?

Reply:
I wasn't sure how the backend handled the payload. Would not appending anything work too?